.github/workflows/bump-version.yaml

```diff
@@ -0,0 +1,96 @@
+# (C) 2023 GoodData Corporation
+name: Bump version & trigger release
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump_type:
+        description: 'Type of version bump to perform (following semver).'
+        type: choice
+        required: true
+        default: 'minor'
+        options:
+          - major
+          - minor
+          - patch
+
+permissions:
+  contents: write
+
+jobs:
+    bump-version:
+      runs-on: ubuntu-latest
+      outputs:
+        new_version: ${{ steps.bump.outputs.new_version }}
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+        - uses: actions/setup-python@v4
+          with:
+            python-version-file: '.python-version'
+            cache: 'pip'
+        - name: Bump version
+          id: bump
+          run: |
+            NEW_VERSION=$(python ./scripts/bump_version.py ${{ github.event.inputs.bump_type }})
+            echo "::set-output name=new_version::$NEW_VERSION"
```

Comment:
This syntax is deprecated and will issue warnings in your output.
It should look like this:
echo "new_version=$NEW_VERSION" >>$GITHUB_OUTPUT